### PR TITLE
Case study folder cleanup

### PR DESCRIPTION
**Pull request by MegumiKim**
As requested I have bundled the case studies covering the same topic under the folders.
Please let me know if it looks good.

### Release Notes

Chore: Removed `case-studies/script-notes-aws-s3.md`.

> "Farewell, old file, you served us well. 
> Your time has come to bid farewell. 
> We'll miss your code and all you've done, 
> But now it's time to move on."

### CODE REVIEW

File: case-studies/webhosting/cPanel-case-study.md
Comment (Fermain) [inline case-studies/webhosting/cPanel-case-study.md:1]: cpanel can live in it's own folder, inside web-hosting

**Comment by Fermain** [comment]
Thanks, this looks good. Just one file to move :)

File: case-studies/webhosting/cPanel-case-study.md
Comment (MegumiKim) [reply case-studies/webhosting/cPanel-case-study.md:1]: Done.
Do you also want folders for those are NOT duplicated?

**Comment by MegumiKim** [comment]
All the case studies now live under the respective folders. 

Some comments:
- There is one script note somehow ended up in the repo. Should I go ahead and delete the file?
- Noticed quite a few simple misspellings in the file "tensorflow-case-study". You might want to ask the next volunteer to fix them?
- I created VPS folder UNDER webhosting. Let me know if you want it otherwise.

File: case-studies/apachecassandra/apache-cassandra-case-study.md
Comment (Fermain) [inline case-studies/apachecassandra/apache-cassandra-case-study.md:1]: this folder can be called `apache`

File: case-studies/aws/AWS-S3-video-case-study.md
Comment (Fermain) [inline case-studies/aws/AWS-S3-video-case-study.md:1]: This could go in `amazon/aws`

File: case-studies/aws/aws-case-study.md
Comment (Fermain) [inline case-studies/aws/aws-case-study.md:1]: Same as above

File: case-studies/googleplatforms/google-platforms-case-study.md
Comment (Fermain) [inline case-studies/googleplatforms/google-platforms-case-study.md:1]: `/google/google-platforms...`

File: case-studies/microsoftazure/microsoft-azure-case-study.md
Comment (Fermain) [inline case-studies/microsoftazure/microsoft-azure-case-study.md:1]: `microsoft/azure`

**Review by Fermain — CHANGES_REQUESTED** [review]
Looking good! just a few more adjustments :)

**Comment by Fermain** [comment]
Where is the script note?

VPS folder is fine thanks

**Comment by MegumiKim** [comment]
Changes reflected!
Script file is called script-notes-aws-s3.md. It lives directly under the case-studies.

**Comment by Fermain** [comment]
You can remove that file :)

**Comment by MegumiKim** [comment]
Done :)

**Review by Fermain — APPROVED** [review]

**Pull request merged**